src/Augurk.UI/wwwroot/css/bootstrap/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Misschien willen we dit ook weer met libman doen? Voel er niet zo heel veel voor om weer heel bootstrap in onze repo te hebben?

Reply:
Agreed

Reply:
Fixed